### PR TITLE
adds media type discussion to envelope and message structure

### PR DESCRIPTION
**Pull request by troyronda**

### CODE REVIEW

**Review by Baha-sk — APPROVED** [review]

File: encryption.md [hunk 1]
```diff
@@ -34,6 +34,12 @@ TODO: Include language about safe nonce considerations.
 
 When the sender wishes to remain anonymous, they should identify themselves as a sender using a newly minted DID that's never been seen before. When the sender is done with the interaction, they can abandon further use of that DID. 
 
+## Media Types
+
+The media type of the envelope is included in the `typ` [property](https://tools.ietf.org/html/rfc7516#section-4.1.11) and the media type of the payload is included in the `cty` [property](https://tools.ietf.org/html/rfc7516#section-4.1.12). The [message types](#message-types) section provides general discussion of the media types.
```
Comment (kdenhartog) [inline encryption.md:39]: Can we switch this to normative language? This looks good otherwise

**Review by kdenhartog — CHANGES_REQUESTED** [review]
see comments below (GH now requires a comment here?)

File: encryption.md (hunk 1 above)
Comment (troyronda) [reply encryption.md]: @kdenhartog I've now added MUST include 'typ' and 'cty' to both the JWE and JWS sections.

**Review by kdenhartog — APPROVED** [review]

**Comment by tplooker** [comment]
Discussed on WG call 23/04/21 resolved to merge

**Pull request merged**